apps/api/src/server/applications/examination-timetable-items/examination-timetable-items.controller.js

```diff
@@ -33,5 +35,34 @@ export const createExaminationTimetableItem = async (_request, response) => {
 	const { body } = _request;
 	const examinationTimetableItem = await exminationTimetableItemsRepository.create(body);
 
+	// find the examination folder id to create subfolder.
+	let examinationFolder = await folderRepository.getFolderByNameAndCaseId(
+		body.caseId,
+		'Examination'
+	);
+
+	if (!examinationFolder) {
+		examinationFolder = await folderRepository.createFolder({
```

Comment:
Since Examination is a system folder, I think it's best we actually just throw an Error here because it probably means the case has not been configured properly. I think we should just create system folders in a single place, because it prevents drift.